.gitignore

```diff
@@ -7,3 +7,6 @@
 .pnp.*
 
 /node_modules
+
+.DS_Store
```

Comment:
https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/

Reply:
Good point, will do it 👍